src/shipit_static_analysis/default.nix

```diff
@@ -5,7 +5,7 @@ let
 
   inherit (releng_pkgs.lib) mkTaskclusterHook mkPython fromRequirementsFile filterSource ;
   inherit (releng_pkgs.pkgs) writeScript makeWrapper fetchurl dockerTools gcc
-      cacert gcc-unwrapped glibc glibcLocales xorg;
+      cacert gcc-unwrapped glibc glibcLocales xorg llvmPackages_4;
```

Comment:
Quick question, are you building `clang-tidy` on top of `LLVM 4.0`? I ask because `LLVM 5.0` was released on September 7th, and I think we should use the more recent version if possible.

Reply:
We are using LLVM 4.0. The 5.0 version is not yet available through standard Nix package.